src/main/sql/add_user_roles.sql

```diff
@@ -0,0 +1,58 @@
+-- This script creates roles which will be be available for assigning to users later.
+--/
+CREATE OR REPLACE SCRIPT row_level_security_test_schema.add_user_role(role_name) returns table AS
+        log = {}
+        table_name = 'EXA_ROLES_MAPPING'
+
+        function add_role_into_new_table()
+                log[#log+1] = {"Table "..table_name.." not found. Creating a new table"}
```

Comment:
```suggestion
                table.insert(log, "Table " .. table_name .. " not found. Creating a new table.");
```
Also for all other occurrences.

Reply:
Please surround all binary operators (those with two arguments) with spaces for better readability and so it looks identical to our Java code.

Reply:
outdated